components/atoms/ChooseImage.tsx

```diff
@@ -0,0 +1,92 @@
+import { Plus } from '@tamagui/lucide-icons'
+import * as ImagePicker from 'expo-image-picker'
+import React, { useState } from 'react'
+import { Alert, useColorScheme } from 'react-native'
+import { Button, Image, XStack } from 'tamagui'
+
```

Comment:
The interface can be optimized: you can add state processing (like loading when downloading photos) to improve user experience.